sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/polling/implementation/PollingConstants.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.util.polling.implementation;
+
+/**
+ * INTERNAL CLASS.
+ *
+ * The constants use for polling related operations.
+ */
+public final class PollingConstants {
```

Comment:
Instead of duplicating all these implementation classes from core, we should consider exporting `com.azure.core.util.polling.implementation` to `azure-core-experimental`. 

Add `exports com.azure.core.util.polling.implementation to com.azure.core.experimental` to azure-core's `module-info.java`.